packages/core/protocols/src/proto/dxos/agent/echo-proxy.proto

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
Other plugins define communication protocols, bigger configs, etc., It will be confusing to place all protobufs for them in one file, I agree that it looks like that specific config does not deserve a separate protobuf file, but it is consistent with general pattern